webdriver/tests/bidi/script/add_preload_script/contexts.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+async def test_add_preload_script_in_iframe_with_top_context_specified(
+        bidi_session, add_preload_script, new_tab,
+        test_page_same_origin_frame):
+
+    await add_preload_script(
+        function_declaration="() => { window.bar='foo'; }",
+        contexts=[new_tab["context"]])
+
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"],
+        url=test_page_same_origin_frame,
```

Comment:
We should test both same and cross origin frames, as from the implementation perspective it's very different at least in Chromium.